netweaver/install_ers.sls

```diff
@@ -39,7 +41,7 @@ netweaver_install_{{ instance_name }}:
     - product_id: NW_ERS:NW750.HDB.ABAPHA
     - cwd: {{ netweaver.installation_folder }}
     - additional_dvds: {{ netweaver.additional_dvds }}
-    - ascs_password: {{ node.master_password }}
+    - ascs_password: {{ netweaver.master_password }}
```

Comment:
Shouldn't this be `{{ netweaver.sid_adm_password|default(netweaver.master_password) }}` ?